README.md

```diff
@@ -20,25 +20,25 @@ Support...
-| Components Type support                    | struct<br/>/class     | struct<br/>/class?    | struct<br/>/class<br/>/interface | struct?            | struct                          | class?                       | struct                     | class                           | class               | class?              | Components Type support                    |
+| Components Type support                    | struct                | struct<br/>/class?    | struct<br/>/class<br/>/interface | struct?            | struct                          | class?                       | struct                     | class                           | class               | class?              | Components Type support                    |
 | Code Generation based                      | :x:                   | :x:                   | :x:                              |                    | :x:                             | :heavy_check_mark:           | :heavy_check_mark:         | :heavy_check_mark:              | :x:                 | :x:                 | Code Generation based                      |
 | Dependencies                               | None                  | None                  | None                             | None               | Unity<br/>Odin Inspector        |                              | Unity                      | Unity                           | Unity               | Unity               | Dependencies                               |
 | Engine agnostic                            | :heavy_check_mark:    | :heavy_check_mark:    | :x:?                             | :heavy_check_mark: | :x:                             | :x:?                         | :x:                        | :x:                             | :x:                 | :x:                 | Engine agnostic                            |
 | Custom Engine Support                      | :heavy_check_mark:    | :heavy_check_mark:    | :heavy_check_mark:               |                    | :heavy_minus_sign:?             | :heavy_minus_sign:           | :x:                        | :x:                             | :x:                 | :x:                 | Custom Engine Support                      |
-| Threading support                          | :heavy_check_mark:    | :heavy_check_mark:Ex  | :heavy_check_mark:               |                    | :x:                             |                              | :heavy_minus_sign:?        | :x:                             | :x:?                | :heavy_minus_sign:? | Threading support                          |
+| Threading support                          | :heavy_check_mark:Ex  | :heavy_check_mark:Ex  | :heavy_check_mark:               |                    | :x:                             |                              | :heavy_minus_sign:?        | :x:                             | :x:?                | :heavy_minus_sign:? | Threading support                          |

```

Comment:
https://github.com/Leopotam/ecs-threads
Multithreading support I would define as systems/logic that helps the user with multithreaded systems, like these.
Again, not fully sure about threading/multithreading definitions here, since I don't use them myself.

Reply:
So MT-systems are supported with the Ex then. Normal systems don't support threading by default, that's why the "Ex" should be there. Aka added with an extension